specification/maps/data-plane/Common/preview/1.0/common.json

```diff
@@ -120,7 +120,7 @@
         }
       }
     },
-    "CoordinatesPair": {
+    "LatLongPair": {
```

Comment:
This is going to be a breaking change for many other Swagger files, including Traffic. Is it a breaking change for Geolocation, which has already been merged?

Reply:
@daniel-rocha I also updated all the other Swaggers (elevation/route/search/traffic) in [the same commit](https://github.com/Azure/azure-rest-api-specs/pull/16162/commits/633a5aacf926901304f91ec420ba7db254d51fdf) so it should be fine :)